packages/reports/tests/acceptance/report-test.js

```diff
@@ -1479,92 +1470,8 @@ module('Acceptance | Navi Report', function(hooks) {
   });
 
   test('Dimension selector', async function(assert) {
-    assert.expect(9);
-
-    const originalFeatureFlag = config.navi.FEATURES.enableRequestPreview;
-
-    config.navi.FEATURES.enableRequestPreview = false;
-
-    await visit('/reports/1');
-
-    assert.deepEqual(
-      await getAllSelected('dimension'),
-      ['Day', 'Property'],
-      'Selected dimensions and time grain initially include "Day" and "Property"'
-    );
-
-    let dimensionItem = await getItem('dimension', 'Operating System');
-
-    assert.ok(dimensionItem.item.querySelector('.fa-plus-circle'), 'An unselected dimension row has a plus icon');
-
-    // Add Dimension
-    await clickItem('dimension', 'Operating System');
-
-    assert.deepEqual(
-      await getAllSelected('dimension'),
-      ['Day', 'Operating System', 'Property'],
-      'Adding a dimension changes selected dimensions'
-    );
-
-    // Add selected dimension as filter
-    await clickItemFilter('dimension', 'Operating System');
```

Comment:
Are we testing adding a dim filter in the test below?

Reply:
That seems to be the goal, would you like me to add those into the test that used to be enabled request preview?